mmv1/third_party/terraform/services/compute/resource_compute_network_test.go.erb

```diff
@@ -111,7 +116,7 @@ func TestAccComputeNetwork_routingModeAndUpdate(t *testing.T) {
 func TestAccComputeNetwork_numericId(t *testing.T) {
 	t.Parallel()
 	suffixName := acctest.RandString(t, 10)
-    networkName := fmt.Sprintf("tf-test-network-basic-%s", suffixName)
+	networkName := fmt.Sprintf("tf-test-network-basic-%s", suffixName)
     projectId := envvar.GetTestProjectFromEnv()
     networkId := fmt.Sprintf("projects/%v/global/networks/%v", projectId, networkName)
```

Comment:
```suggestion
	networkName := fmt.Sprintf("tf-test-network-basic-%s", suffixName)
	projectId := envvar.GetTestProjectFromEnv()
	networkId := fmt.Sprintf("projects/%v/global/networks/%v", projectId, networkName)
```

Reply:
(I know you didn't add these lines in this PR, but it's good to tidy up code as you go!)

Reply:
the render whitespace is nice to have, It's always annoying to find it later on so this should prevent that from happening!